tensorflow/lite/kernels/arg_min_max.cc

```diff
@@ -37,7 +37,12 @@ constexpr int kOutputTensor = 0;
 
 TfLiteStatus ResizeOutput(TfLiteContext* context, const TfLiteTensor* input,
                           const TfLiteTensor* axis, TfLiteTensor* output) {
-  int axis_value = *GetTensorData<int>(axis);
+  int axis_value;
+  if (axis->type == kTfLiteInt64) {
+    axis_value = *GetTensorData<int64_t>(axis);
```

Comment:
Could you make it explicit? Probably by wrapping this up with static_cast<int> or something.

Reply:
Sure, `static_cast` should work for this one.